src/main/java/com/example/bulletinboardapp/controller/CommentController.java

```diff
@@ -0,0 +1,34 @@
+package com.example.bulletinboardapp.controller;
+
+import com.example.bulletinboardapp.entity.Comment;
+import com.example.bulletinboardapp.dto.Comments;
+import com.example.bulletinboardapp.service.impl.CommentServiceImpl;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/ads")
+public class CommentController {
+    private final CommentServiceImpl commentService;
```

Comment:
```suggestion
    private final CommentService commentService;
```
инъекцию надо делать по интерфейсам, а не реализации